### PR TITLE
Add vignette

### PR DESCRIPTION
**Pull request by miguelmorin**
This pull request is stale because the Travis checks are not running. We moved the commits to #58  and we can now close this pull request. We opened a thread on [DevOps StackEchange](https://devops.stackexchange.com/questions/8000/github-and-travis-integration-broke-down-for-a-branch) and [a ticket with Travis](https://travis-ci.community/t/github-integration-broken-for-a-branch-with-pull-request/3457) to ask why the checks were not running.

### CODE REVIEW

File: vignettes/anpl.Rmd [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Vignette Title"
-author: "Vignette Author"
+title: "Adaptive non-parametric learning"
+author: "Simon Lyddon, Miguel Morin, and James Robinson"
 date: "`r Sys.Date()`"
```
Comment (jemrobinson) [inline vignettes/anpl.Rmd]: Should this be today's date?

Comment (miguelmorin) [reply vignettes/anpl.Rmd]: No, the standard is to have the date at which the vignette is run and built, inserted automatically with that command.

**Comment by jemrobinson** [comment]
The code here was included in #58. This ticket was left open for debugging why the Travis checks were not running, but it's not clear we're going to get an answer to this. Closing.

**Comment by miguelmorin** [comment]
Closing this ticket as it's unlikely to be debugged. If needed, we can reopen it and the problem of missing Travis checks will persist.

**Pull request closed without merging**